client/lib/wporg/index.js

```diff
@@ -32,7 +33,8 @@ async function getRequest( url, query ) {
 	if ( response.ok ) {
 		return await response.json();
 	}
-	throw new Error( await response.body );
+
+	throw new RequestError( await response.body, response );
```

Comment:
The keyword `await` here is not required as `response.body` is not asynchronous

Reply:
Yes, I agree. But I intentionally decided to keep it as is to avoid breaking changes on places where it is already being used this way.